Library/Homebrew/compat/requirements/minimum_macos_requirement.rb

```diff
@@ -0,0 +1,3 @@
+require "requirements/macos_requirement"
```

Comment:
Stick this file's contents in `homebrew/Library/Homebrew/compat/requirements.rb`